Cargo.toml

```diff
@@ -37,4 +37,4 @@ rand_chacha = "0.3"
 default = ["std"]
 nightly = []
 debug-transcript = ["hex"]
-std = ["rand_core/std", "byteorder/std"]
+std = ["rand_core/alloc", "byteorder/std"]
```

Comment:
Why not adding `alloc` feature instead? You still have `byteorder/std` so it doesn't really make a big difference, you'll still get `std` with this change